src/tests/ftest/pool/create_capacity.yaml

```diff
@@ -21,6 +21,10 @@ server_config:
       scm_class: dcpm
       scm_list: ["/dev/pmem0"]
       scm_mount: /mnt/daos0
+      log_mask: DEBUG
```

Comment:
are we sure this wont result in giant logs on every pr? we try to not set mask to debug unless absolutely needed due to artifact size constraints

Reply:
For this test it appears to be low magnitude of log file sizes & lower risk. For example master build 6359 https://build.hpdd.intel.com/job/daos-stack/job/daos/job/master/6359/artifact/Functional%20Hardware%20Large/pool/create_capacity.py/ , versus this PR 8279 build 2 https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8279/2/artifact/Functional%20Hardware%20Large/pool/create_capacity.py/ is 12MB of storage versus 63MB of storage (and for the latter I took all of the daos_logs/*.bz2 files and uncompressed them for the comparison)

I also looked at the daos_test/suite.py folders after downloading from a recent master 6341 and this PR 8279 build 2. The uncompressed daos_logs comparison is 1.9G vs 2.0G of space (the change in this PR is not to the log level, but just the DD_MASK=any). So daos_test tends to log quite a bit more than pool/create_capacity.py test due to both its DEBUG log level and running a lot more tests.

Reply:
while this is good for the passing tests, i am more worried about logs getting out of hand when the failure happens and debug is on. Is there a way to compare somehow previously failing sizes with sizes that would be now if test was to fail in the same manner?

Reply:
I have some experience with a failing pool/create_capacity.py scenario in https://daosio.atlassian.net/browse/DAOS-9108
While there were no artifacts attached to that ticket, I do have some saved on my machine as I conducted the investigation.

In one failed test instance from master build 5878 (with log_mask: INFO i.e., without the proposed DEBUG log mask change from this PR - and with provider ofi+verbs;ofi_rxm) I found the create_capacity.py artifacts with uncompressed server/engine logs to take 22M of storage. Another create_capacity.py artifact (that I believe is from the master build 5824 from the original problem report) occupies 26M of storage. It is not practical at this time to recreate those failures with DEBUG log mask enabled.